easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,480 @@
+    }),
+    ('setuptools_scm', '6.4.2', {
+        'checksums': ['6833ac65c6ed9711a4d5d2266f8024cfa07c533a0e55f4c12f6eff280a5a9e30'],
+    }),
+    ('python-dateutil', '2.8.2', {
+        'modulename': 'dateutil',
+        'checksums': ['0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86'],
+    }),
+    ('decorator', '5.1.1', {
+        'checksums': ['637996211036b6385ef91435e4fae22989472f9d571faba8927ba8253acbc330'],
+    }),
+    ('liac-arff', '2.5.0', {
+        'modulename': 'arff',
+        'checksums': ['3220d0af6487c5aa71b47579be7ad1d94f3849ff1e224af3bf05ad49a0b5c4da'],
+    }),
+    ('pycrypto', '2.6.1', {
+        'modulename': 'Crypto',
+        'patches': ['pycrypto-2.6.1_remove-usr-include.patch'],
+        'checksums': [
+            'f2ce1e989b272cfcb677616763e0a2e7ec659effa67a88aa92b3a65528f60a3c',  # pycrypto-2.6.1.tar.gz
+            # pycrypto-2.6.1_remove-usr-include.patch
+            '06c3d3bb290305e1360a023ea03f9281116c230de62382e6be9474996086712e',
+        ],
+    }),
+    ('ecdsa', '0.17.0', {
+        'checksums': ['b9f500bb439e4153d0330610f5d26baaf18d17b8ced1bc54410d189385ea68aa'],
+    }),
+    ('ipaddress', '1.0.23', {
+        'checksums': ['b7f8e0369580bb4a24d5ba1d7cc29660a4a6987763faf1d8a8046830e020e7e2'],
+    }),
+    ('asn1crypto', '1.5.1', {
+        'checksums': ['13ae38502be632115abf8a24cbe5f4da52e3b5231990aff31123c805306ccb9c'],
+    }),
+    ('idna', '3.3', {
+        'checksums': ['9d643ff0a55b762d5cdb124b8eaa99c66322e2157b69160bc32796e824360e6d'],
+    }),
+    ('pycparser', '2.21', {
+        'checksums': ['e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206'],
+    }),
+    ('cffi', '1.15.0', {
+        'checksums': ['920f0d66a896c2d99f0adbb391f990a84091179542c205fa53ce5787aff87954'],
+    }),
+    ('semantic_version', '2.9.0', {
+        'checksums': ['abf54873553e5e07a6fd4d5f653b781f5ae41297a493666b59dcf214006a12b2'],
+    }),
+    ('typing_extensions', '4.2.0', {
+        'checksums': ['f1c24655a0da0d1b67f07e17a5e6b2a105894e6824b92096378bb3668ef02376'],
+    }),
+    ('setuptools-rust', '1.3.0', {
+        'checksums': ['958c5bf4ab6483d59dab888538121871cc5006354a42fb0fbd50acf03caad1de'],
+    }),
+    ('cryptography', '37.0.1', {
+        # avoid that cargo uses $HOME/.cargo, which can lead to build failures if home directory is NFS mounted,
+        # see https://github.com/rust-lang/cargo/issues/6652
+        'preinstallopts': "export CARGO_HOME=%(builddir)s/cargo && ",
+        'checksums': ['d610d0ee14dd9109006215c7c0de15eee91230b70a9bce2263461cf7c3720b83'],
+    }),
+    ('pyasn1', '0.4.8', {
+        'checksums': ['aef77c9fb94a3ac588e87841208bdec464471d9871bd5050a287cc9a475cd0ba'],
+    }),
+    ('PyNaCl', '1.5.0', {
+        'modulename': 'nacl',
+        'checksums': ['8ac7448f09ab85811607bdd21ec2464495ac8b7c66d146bf545b0f08fb9220ba'],
+    }),
+    ('bcrypt', '3.2.0', {
+        'checksums': ['5b93c1726e50a93a033c36e5ca7fdcd29a5c7395af50a6892f5d9e7c6cfbfb29'],
```

Comment:
```suggestion
    ('bcrypt', '3.2.2', {
        'checksums': ['433c410c2177057705da2a9f2cd01dd157493b2a7ac14c8593a16b3dab6b6bfb'],
```

Reply:
They've released two new versions (3.2.1 and 3.2.2) since I did this yesterday!